src/components/CustomerQuota/utils.test.ts

```diff
@@ -394,11 +402,11 @@ describe("pod related utils", () => {
         ).toStrictEqual(true);
       });
 
-      it("should return false for non-pod category", () => {
+      it("should return true for any category", () => {
         expect.assertions(1);
         expect(
           isPodChargeable(idType, validIdentifiers, validCartItem)
```

Comment:
oh just noticed this, whats the difference between this test and the test for tt-token category? on line 397

Reply:
the new one will have "toilet-paper" as the category!